csunplugged/templates/base.html

```diff
@@ -27,7 +27,25 @@
     </title>
     <meta name="description" content="">
     <meta name="author" content="">
+
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+
+
+
+    <script src="{% static 'js/codemirror.js' %}"></script>
```

Comment:
All these scripts should only be loaded on the pages that require them, rather than adding this overhead to every page.